tests/accounts/test_views.py

```diff
@@ -167,3 +263,27 @@ def test_post_forgets_user(self, monkeypatch, pyramid_request):
         assert result.headers["foo"] == "bar"
         assert forget.calls == [pretend.call(pyramid_request)]
         assert pyramid_request.session.invalidate.calls == [pretend.call()]
+
+    def test_post_redirects_user(self, monkeypatch, pyramid_request):
+        forget = pretend.call_recorder(lambda request: [("foo", "bar")])
```

Comment:
You also don't need to worry about testing that the logout view handles the ``foo: bar`` header, since that's tested elsewhere.